stdlib/ballerina-websub/src/main/java/org/ballerinalang/net/websub/BallerinaWebSubConnectionListener.java

```diff
@@ -272,6 +243,15 @@ private void autoRespondToIntentVerification(HTTPCarbonMessage httpCarbonMessage
                 response.waitAndReleaseAllEntities();
                 URIUtil.populateQueryParamMap(queryString, params);
                 String mode = params.get(WebSubSubscriberConstants.PARAM_HUB_MODE).stringValue();
+                if (!params.keySet().contains(WebSubSubscriberConstants.PARAM_HUB_MODE) ||
```

Comment:
Can't we do a static import for WebSubSubscriberConstants ?

Reply:
Fixed all occurrences in this class with 35aba7f